main.lua

```diff
@@ -0,0 +1,104 @@
+imgxKaizer=200
+imgyKaizer=0
+
+imgxC1 = 0
+imgyC1 = 100
+
+imgxC2 = 0
+imgyC2 = 200
+
+imgxC3 = 0
+imgyC3 = 300
+
+function love.load()
+    imageKaizer = love.graphics.newImage("Kaizer.JPG")

```

Comment:
Consider placing your images into a separate assets folder

Reply:
Images placed in separate assets folder and linked accordingly.